cobalt/browser/cobalt_content_browser_client.cc

```diff
@@ -48,8 +49,8 @@ class CobaltBrowserMainParts : public content::ShellBrowserMainParts {
 
   // ShellBrowserMainParts overrides.
   int PreCreateThreads() override {
-    // TODO(b/372559349): setup metrics similarly to what SetupMetrics() does
-    // when called from ChromeBrowserMainParts::PreCreateThreadsImpl().
+    metrics_ = std::make_unique<CobaltMetricsServiceClient>();
+    metrics_->Start();
     return ShellBrowserMainParts::PreCreateThreads();
```

Comment:
Would you do me a favor and add a stubbed out version of `PreMainMessageLoopRun` to CobaltBrowserMainParts and add a TODO to call `StartMetricsRecording`? `PreMainMessageLoopRun` should already be on ShellBrowserMainParts, just need to call super: https://github.com/youtube/cobalt/blob/8d8a8ddd7b1459b354ac0187201e3281a44e3d05/ios/web/shell/shell_web_main_parts.h#L24.